src/screens/ChildrenScreen.js

```diff
@@ -1,158 +1,126 @@
 import React from "react";
-import {
-  ScrollView,
-  TouchableOpacity,
-  StyleSheet,
-  View,
-  Text,
-} from "react-native";
-import { Ionicons } from "@expo/vector-icons";
+import { Image, StyleSheet, View, Text, SafeAreaView } from "react-native";
 
-import colors from "../config/colors";
+import { AuthContext } from "../config/context";
+import { scale, moderateScale, verticalScale } from "../config/scaling";
 import screen from "../config/screen";
+import colors from "../config/colors";
+import { TouchableOpacity } from "react-native-gesture-handler";
+import { Card } from "react-native-elements";
+import { Ionicons } from "@expo/vector-icons";
+import AddChildScreen from "./AddChildScreen";
```

Comment:
Erase the modules you don't use on this screen. One of the modules you don't use creates an error.
![image](https://user-images.githubusercontent.com/45149474/118330973-1e57be80-b4d6-11eb-8419-b0ee6f0c8a2f.png)